src/vm/nativelibrarynative.h

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// 
+// File: NativeLibraryNative.h
+//
+
+//
+// QCall's for the NativeLibrary class
```

Comment:
Superfluous.

Reply:
I agre; but I followed the existing convention in other native.h files https://github.com/dotnet/coreclr/blob/master/src/vm/marshalnative.h#L9
https://github.com/dotnet/coreclr/blob/master/src/vm/assemblynative.hpp#L9